src/classes/BDI_DataImportService.cls

```diff
@@ -72,6 +72,27 @@ global with sharing class BDI_DataImportService {
         return bdiMS;
     }
 
+    /*******************************************************************************************************
+    * @description constructs the soql String to run a Data Import process for list of DataImport ids.
+    * @param dataImportIds list of data import ids to include in query
+    * @return String the soql String
+    */
+    public static String strSoqlForDataImportProcess(List<Id> dataImportIds) {
+        String dataImportQuery;
+
+        // construct or SOQL for the batch
+        dataImportQuery = 'SELECT ' + String.join(listStrDataImportFields, ',');
+        dataImportQuery += ' FROM ' + DataImport__c.SObjectType.getDescribe().localName;
```

Comment:
I would suggest one of two strategies here: we can either use the describe result to get the name like this, but maybe go through our `UTIL_Describe` class to get the benefit of caching, or `String.valueOf(DataImport__c.SObjectType)` to avoid the call to get the describe.  
Either way, I think we specifically want to avoid `localName` as it does NOT include the namespace even when in a namespaced context:
https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_methods_system_sobject_describe.htm

Reply:
Thanks for calling that out @kenlewis-sfdo